access/src/main/external/static/recordTransformations/modsToFullRecord.xsl

```diff
@@ -544,15 +549,15 @@
 		</xsl:for-each-group>
 	</xsl:template>
 
-	<!-- mods:location -->
+	<!-- mods:location  p.s. why is there uri here?  that isn't a valid mods element. sn March 2015 -->
```

Comment:
It looks like a pretty good element though doesn't it?  Yeah, I must have spontaneously generated that one.  Can you remove it as part of this pull request?